salesagility/SuiteCRM
# Add a popup to confirm leaving emails compose view modal

Adds a confirmation dialog when clicking away the emails compose view modal to warn the user about losing form data.

## Motivation and Context

Fixes #3778 

## How To Test This


1. Repair and Rebuild, clear JS cache
2. Open emails compose view modal from subpanel or listview bulk action
3. Click outside the modal to close it
4. See a confirmation dialog appears.

## Types of changes


### Final checklist
